ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -13,7 +13,9 @@ CREATE TABLE parameters (
   ledger_end_sequential_id BIGINT NOT NULL,
   ledger_end_string_interning_id INTEGER NOT NULL,
   participant_pruned_up_to_inclusive VARCHAR,
-  participant_all_divulged_contracts_pruned_up_to_inclusive VARCHAR
+  participant_all_divulged_contracts_pruned_up_to_inclusive VARCHAR,
+  ledger_metering_end VARCHAR,
```

Comment:
I sort of expected we would only care about timestamps as aggregating. Why do we need to add offset-s in the mix?
just the plain fact that due to parallel ingestion the offset ordered transaction_metering records would not be monotonically increasing in terms of timestamps makes me uneasy with the queries, and the aggregation logic below

Reply:
The answer to this is in the design spec transaction metering appendix I can go through with you.